app/config/settings.py

```diff
@@ -640,6 +640,9 @@ def strtobool(val) -> bool:
 )
 # The name of the group whose members will be able to create workstations
 WORKSTATIONS_CREATORS_GROUP_NAME = "workstation_creators"
+WORKSTATIONS_MAXIMUM_TIMEOUT = int(
+    os.environ.get("WORKSTATIONS_MAXIMUM_TIMEOUT", "10000")
```

Comment:
If I understand correctly, this (and the equally named environment variable) will enforce a maximum of 10.000 seconds (~2,78 hours). I think for my use case it will be better if the max is >8 hours. As the graders will probably want to use the workstation for a full working day without needing to restart the session.

Reply:
This is why I made it an environment variable, we can set that in production for our use case :-)